packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -198,9 +226,45 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                 }
             },
             // eth addresses are valid without checksum but Trezor displays them as checksummed
-            checksum: (value: string) => {
-                if (networkType === 'ethereum' && !checkAddressCheckSum(value)) {
-                    return translationString('RECIPIENT_IS_NOT_VALID');
+            checksum: async (address: string) => {
+                if (networkType === 'ethereum') {
+                    const updatedValues = addValue(address); // Get updated values immediately
+                    const params = {
+                        descriptor: address,
+                        coin: symbol,
+                    };
+
+                    //they are not the same but the second one is checksum of the first one, in this case show the info that the address was automatically checksummed
+                    if (
+                        updatedValues &&
+                        updatedValues[0] &&
+                        updatedValues[1] &&
+                        updatedValues[0] !== updatedValues[1] &&
+                        toChecksumAddress(updatedValues[0]) === updatedValues[1]
+                    ) {
+                        setAddressChecksumInfo({
+                            text: 'Address converted to checksum format.',
+                        });
+                    } else {
+                        setAddressChecksumInfo(undefined);
+                    }
```

Comment:
Why is this required? What if we just put 
this
```
                 setAddressChecksumInfo({
                            text: 'Address converted to checksum format.',
                        });
```
here

```
if (result.success && result.payload.history.total !== 0) {
                        if (!checkAddressCheckSum(address)) {
                            setValue(inputName, toChecksumAddress(address), {
                                shouldValidate: true,
                            });
                            setAddressChecksumInfo({
                                text: 'Address converted to checksum format.',
                            });
                        }
                    }
```

Reply:
You are actually right. 

The problem was when the address was changed through toChecksumAddress(), it also triggered the validation logic again and the information bubble disappeared. That why I introduced that non very elegant logic. To be completely honest I didn't know that onChange is triggered only when the input is changed through the keyboard or paste, but not when it changed programatically with the toChecksum...() func.  This solves everything and that logic is not needed.